README.md

```diff
@@ -0,0 +1,43 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Portfolio
+
+> To do List
+
```

Comment:
- [ ] I think you have forgotten to change the title name of your project since it is not port, Kidly change this and r

- [ ]  Your README looks great 👍🏻  but I think you can improve it by making it more professional. A professional README file should have a project description.

-  [ ] Kindly, Add a descriptive README file to your project, it will be great to add instructions on how to run the project. This includes `npm install`, `npm start` etc.  Please refer to this [template](https://github.com/microverseinc/readme-template), you can copy and paste the content but remember to customize it to your project.